### DISCUSSION TITLE
How can I target a specific version?

### DISCUSSION THREAD

**Discussion by smholvoet** [squidfunk/mkdocs-material#3374] [Q&A] [answered]
Might be a silly question, but I failed to find this in the current docs.

My `mkdocs.yml` has the following section, which doesn't  specify a version.

```yaml
...
theme:
  name: material
...
```

Does this mean it is always pointing to the **latest** release (which is fine by me)?

FYI: I'm using the Docker image to set up my docs locally before it gets built and pushed to an Azure Static Web App.

**Comment by squidfunk** [comment, marked as answer]
You need to specify the version you want to use during installation:

When installing via `pip`:

```
pip install mkdocs-material==8.1.3
```

When installing via `docker`:

```
docker pull squidfunk/mkdocs-material:8.1.3
```

**Reply by smholvoet** [reply]
Ok, that makes sense.

I might explicitly specify `pip install mkdocs-material==<version number>` in my build pipeline to make it clearer which exact version of the theme I'm  currently on.

Thanks for the quick reply and awesome work! 🚀

**Reply by squidfunk** [reply]
I made a note in #3363 that we add this information to the docs, and yes, specifying a dedicated version and upgrading deliberately is a good idea when deploying documentation automatically to production 😊
